mycpp/gc_builtins.h

```diff
@@ -59,6 +63,22 @@ inline RuntimeError::RuntimeError(Str* message)
     : Obj(Tag::FixedSize, maskof_RuntimeError(), kNoObjLen), message(message) {
 }
 
+// libc::wcswidth raises UnicodeError
+class UnicodeError : public Obj {
+ public:
+  explicit UnicodeError(Str* message);
+  Str* message;
+
+  static constexpr uint16_t field_mask() {
+    return maskbit(offsetof(UnicodeError, message));
+  }
+};
+
+inline UnicodeError::UnicodeError(Str* message)
```

Comment:
minor: if the constructor is in the header, we should put them in the `class { }`

The reason there are some constructors outside `class { }` was because I thought we needed to do that for field masks!    We couldn't define the mask function without the compiler saying it hadn't seen the fields yet.

But I think you showed otherwise, so I want to go back and change the style everywhere!

Reply:
ah great. i was wondering why the surrounding code was like this. will send a follow up to make them consistent with the new style